ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -622,14 +622,16 @@ namespace OrdinalAxis {
         ): number {
             const ordinal = this,
                 axis = ordinal.axis,
+                min = axis.min as any,
```

Comment:
Avoid `any` casts in new code. `axis.min` is undefined when there are not data points in series related to the axis. What happens with this logic in such case?

Reply:
> What happens with this logic in such case?

Probably the same as we do with non-ordinal axes: NaN's ([demo](https://jsfiddle.net/BlackLabel/xmbc5p38/)) 🤔